crates/bevy_render/src/camera/projection.rs

```diff
@@ -156,6 +157,7 @@ impl Default for OrthographicProjection {
             window_origin: WindowOrigin::Center,
             scaling_mode: ScalingMode::WindowSize,
             scale: 1.0,
+            depth_calculation: DepthCalculation::Distance,
```

Comment:
`Distance` makes sense as the default, because it always works everywhere. The 2D camera bundles will override it with `ZDifference`.